drivers/rtc/rtc_pcf2123.c

```diff
@@ -0,0 +1,663 @@
+/**
+ * Copyright (c) 2025 Lukas Gunnarsson <lukasgunnarsson@protonmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Supported features:
+ * - Base RTC functionality (set/get time)
+ * - Alarm interrupts
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/drivers/rtc.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/timeutil.h>
+
+#include <stdint.h>
+#include <string.h>
+#include <sys/errno.h>
+
+#include "rtc_utils.h"
```

Comment:
Probably a copy paste issue, but you only need:
```
#include <zephyr/kernel.h>
#include <zephyr/drivers/spi.h>
#include <zephyr/drivers/rtc.h>
#include <zephyr/sys/util.h>
#include <zephyr/logging/log.h>
#include "rtc_utils.h"
```
from what I can see in the driver :)